src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -177,6 +180,50 @@ export class DotEditContentHtmlService {
         this.currentAction = DotContentletAction.EDIT;
     }
 
+    /**
+     * Set the same height to containers in the same row
+     *
+     * @param {DotLayout} pageLayout
+     * @memberof DotEditContentHtmlService
+     */
+    setContaintersSameHeight(pageLayout: DotLayout): void {
+        const doc = this.getEditPageDocument();
+        const rowsMaxHeight: number[] = [];
+        let rowHeight: number;
+
+        const containersLayoutIds = pageLayout.body.rows.map((row: DotLayoutRow) => {
+            return row.columns.map((column: DotLayoutColumn) => {
+                return {
+                    identifier: column.containers[0].identifier,
+                    uuid: column.containers[0].uuid
+                };
+            });
+        });
+
+        const containerDomElements = containersLayoutIds.map((containerRow: Array<DotPageContainer>, index: number) => {
+            rowHeight = 0;
+            return containerRow.map((container: DotPageContainer) => {
+                const querySelector = [
+                    `div[data-dot-object="container"]`,
+                    `[data-dot-identifier="${container.identifier}"]`,
+                    `[data-dot-uuid="${container.uuid}"]`
+                ].join('');
+                const containerElement = doc.querySelector(querySelector);
+                containerElement.style.height = 'auto';
```

Comment:
Why do we need this auto here? don't we overwrite this in the line 222?

Reply:
this is to restore the `height` of the containers, when you first load the page this is not needed, but then when you start doing modifications then the height of the containers start changing and you need to restore it to the original-height of each container to get the tallest one